### PR TITLE
Run CI for 1.2.0 serving, net-istio and net-kourier release

### PR DESCRIPTION
**Pull request by skonto**
As per title, needed for Serverless 1.23.

/cc @nak3 @rhuss

### CODE REVIEW

File: ci-operator/config/openshift-knative/net-istio/openshift-knative-net-istio-release-1.2.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    tag: base
+  openshift_release_golang-1.17:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
```
Comment (skonto) [inline ci-operator/config/openshift-knative/net-istio/openshift-knative-net-istio-release-1.2.yaml:14]: This has to be 1.16 or 1.17? In previous versions we have this as 1.16.

**Comment by skonto** [comment]
```
error: error copying source path /go/bin/volumes to .: exit status 1
```
Updated openshift/serving release-v1.2.0 branch with pvc support.
/retest

**Comment by skonto** [comment]
```
+ oc create namespace istio-system
The connection to the server api.ci-ocp-4-10-amd64-aws-us-east-1-pfmkr.hive.aws.ci.openshift.org:6443 was refused - did you specify the right host or port?
+ for ns in '"${namespaces[@]}"'
```

File: ci-operator/config/openshift-knative/net-istio/openshift-knative-net-istio-release-1.2.yaml (hunk 1 above)
Comment (matzew) [reply ci-operator/config/openshift-knative/net-istio/openshift-knative-net-istio-release-1.2.yaml:14]: IMO would be nice if we align, no ?

File: ci-operator/config/openshift-knative/net-kourier/openshift-knative-net-kourier-release-1.2.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    tag: base
+  openshift_release_golang-1.17:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
```
Comment (matzew) [inline ci-operator/config/openshift-knative/net-kourier/openshift-knative-net-kourier-release-1.2.yaml:14]: but this is 1.16 too ...

File: ci-operator/config/openshift-knative/net-istio/openshift-knative-net-istio-release-1.2.yaml (hunk 1 above)
Comment (skonto) [reply ci-operator/config/openshift-knative/net-istio/openshift-knative-net-istio-release-1.2.yaml:14]: yes

File: ci-operator/config/openshift-knative/net-kourier/openshift-knative-net-kourier-release-1.2.yaml (hunk 2 above)
Comment (skonto) [reply ci-operator/config/openshift-knative/net-kourier/openshift-knative-net-kourier-release-1.2.yaml:14]: yes relic from the previous setup will update

Comment (skonto) [reply ci-operator/config/openshift-knative/net-kourier/openshift-knative-net-kourier-release-1.2.yaml]: @nak3 do we care about it? I just updated to 1.17 any concerns?

**Comment by skonto** [comment]
infra

**Comment by skonto** [comment]
```
   autoscale_hpa_test.go:74: Error collecting metrics by HPA: timed out waiting for the condition
--- FAIL: TestHPAAutoscaleUpDownUp (907.59s)
FAIL test/e2e.TestHPAAutoscaleUpDownUp (907.59s)
FAIL test/e2e

=== Skipped
=== SKIP: test/e2e TestHPAAutoscaleUpDownUpMem (0.00s)
    autoscale_hpa_test.go:64: #11944: Skipped because of excessive flakiness

=== Failed
=== FAIL: test/e2e TestHPAAutoscaleUpDownUp (907.59s)
```

**Comment by skonto** [comment]
```
{"type":"InstallSucceeded","status":"False","lastTransitionTime":"2022-04-07T19:22:54Z","reason":"Error","message":"Install failed with message: no matches for kind \"PodDisruptionBudget\" in version \"policy/v1\""},
```

File: ci-operator/config/openshift-knative/net-kourier/openshift-knative-net-kourier-release-1.2.yaml (hunk 2 above)
Comment (skonto) [reply ci-operator/config/openshift-knative/net-kourier/openshift-knative-net-kourier-release-1.2.yaml]: seems no

**Comment by skonto** [comment]
@matzew pls approve.

**Pull request merged**